packages/@lwc/shared/src/aria.ts

```diff
@@ -66,28 +73,51 @@ const AriaPropertyNames = [
     'role',
 ] as const;
 
+// The list includes prop->attr reflections that we have added in the past,
+// but which are not part of AOM ARIA reflection as supported in browsers.
+// https://github.com/salesforce/lwc/issues/2733
```

Comment:
maybe we can try the other way around. The current implementation does not include these reflections, but keeping the polyfill and the fact this removing is causing failures potentially supports use case to expand the standard.

I want to include the Element reference to this work.

Reply:
@leobalter I mean, I'm perfectly happy if we can convince browsers to implement what we already have. 😄 

AFAICT it's compatible with the `*Element` proposal. I.e.:

```js
element.setAttribute('aria-describedby', 'foo')
element.ariaDescribedBy // 'foo'
element.ariaDescribedByElement // <div id="foo">
```

Similarly for `*Elements`:

```js
element.setAttribute('aria-activedescendant', 'foo bar')
element.ariaActiveDescendant // 'foo bar'
element.ariaActiveDescendantElements // [<div id="foo">, <div id="bar">]
```

Reply:
Yes! I think this will provide us a better value than just the cross-root aria delegation. That one works well in a declarative (html) format, but this one seems like delivering a broader coverage using JS underneath the components.

Reply:
@leobalter I chatted with @caridy about this and we agree that it would be great to get the following reflections into the spec:

- `ariaActiveDescendant` <-> `aria-activedescendant`
- `ariaControls` <-> `aria-controls`
- `ariaDescribedBy` <-> `aria-describedby`
- `ariaDetails` <-> `aria-details`
- `ariaErrorMessage` <-> `aria-errormessage`
- `ariaFlowTo` <-> `aria-flowto`
- `ariaLabelledBy` <-> `aria-labelledby`
- `ariaOwns` <-> `aria-owns`

I think we can argue that this would improve custom element support even outside of LWC, because it harmonizes the attr-prop reflections across all aria attributes, and reduces confusion about how to handle attrs vs props when passing them to custom elements (especially since these are irregular, e.g. `ariaDescribedBy`, not `ariaDescribedby`).

Let me know how I can help with next steps.